cmd/provision/provision.go

```diff
@@ -294,6 +304,31 @@ func (p *provision) run(printf shared.FormatFn) error {
 	return verifyErrors
 }
 
+// getRuntimeType fetches the organization information from the management base and extracts the runtime type
+func (p *provision) getRuntimeType() error {
```

Comment:
Let's rename this function since getXXX() usually implies returning a value. Maybe "retrieveRuntimeType()" or similar?